src/components/TokenSelect/index.tsx

```diff
@@ -13,16 +15,27 @@ import { ThemedText } from 'theme'
 
 import Column from '../Column'
 import Dialog, { Header } from '../Dialog'
-import { inputCss, StringInput } from '../Input'
 import Row from '../Row'
 import Rule from '../Rule'
 import NoTokensAvailableOnNetwork from './NoTokensAvailableOnNetwork'
 import TokenButton from './TokenButton'
 import TokenOptions, { TokenOptionsHandle } from './TokenOptions'
 import TokenOptionsSkeleton from './TokenOptionsSkeleton'
 
-const SearchInput = styled(StringInput)`
-  ${inputCss}
+const SearchInputContainer = styled(Row)`
+  background-color: ${({ theme }) => theme.module};
+  border: 1px solid ${({ theme }) => theme.outline};
+  border-radius: ${({ theme }) => theme.borderRadius}em;
+  justify-content: flex-start;
+  padding: calc(0.75em - 1px);
+
+  &:focus-within {
+    border-color: ${({ theme }) => theme.accentSoft};
+  }
+`
+
+const StyledSearchIcon = styled(Search)`
```

Comment:
This shouldn't be necessary. Instead, use it directly as `<SearchIcon color="secondary" />`.
If that doesn't work, change the wrapping in src/icons/index.tsx so that the color is applied correctly.